internal/mithril-metric/src/metric.rs

```diff
@@ -76,6 +76,65 @@ impl MetricCollector for MetricCounter {
     }
 }
 
+/// Metric counter vec
+pub struct MetricCounterVec {
```

Comment:
Maybe the name could be more explicit:
```suggestion
pub struct MetricCounterWithLabels {
```

Reply:
Done ✅